src/legacy/core_plugins/vis_type_vislib/public/vislib/helpers/point_series/_get_point.ts

```diff
@@ -17,9 +17,46 @@
  * under the License.
  */
 
-import { getFormat } from '../../visualize/loader/pipeline_helpers/utilities';
+import { getFormatService } from '../../../services';
+import { Aspect } from './point_series';
+import { Table, Row } from '../../types';
 
-export function getPoint(table, x, series, yScale, row, rowIndex, y, z) {
+type RowValue = number | 'NaN';
+interface Raw {
+  table: Table;
+  column: number;
+  row: number;
+  value: RowValue;
+}
+export interface Point {
+  x: RowValue | '_all';
+  y: RowValue;
+  z?: RowValue;
+  extraMetrics: [];
+  seriesRaw?: Raw;
+  xRaw: Raw;
+  yRaw: Raw;
+  zRaw?: Raw;
+  tableRaw?: {
+    table: Table;
+    column: number;
+    row: number;
+    value: number;
+    title: string;
+  };
+  parent: Aspect | null;
+  series?: string;
+  seriesId?: string;
+}
+export function getPoint(
+  table: Table,
+  x: Aspect,
+  series: Aspect[] | undefined,
```

Comment:
Why is `yScale` gone?

Reply:
https://github.com/elastic/kibana/pull/61575#discussion_r402888666